packages/bundler/package.json

```diff
@@ -38,7 +38,11 @@
     "!dist/test/**"
   ],
   "dependencies": {
+<<<<<<< HEAD
+    "@typespec/compiler": "~0.45.2",
+=======
     "@typespec/compiler": "~0.45.1",
+>>>>>>> e3a9d8bbf7cda2ce3aabf4142e61df811017dd30
```

Comment:
Merge conflicts remain

Reply:
yeah had a bad merge, should be fixed now